shims/spark320/src/main/scala/com/nvidia/spark/rapids/shims/spark320/Spark320Shims.scala

```diff
@@ -1007,4 +1007,9 @@ class Spark320Shims extends Spark32XShims {
     kryo.register(classOf[SerializeBatchDeserializeHostBuffer],
       new KryoJavaSerializer())
   }
+
+  override def getCentralMomentDivideByZeroEvalResult(): Expression = {
+    val nullOnDivideByZero: Boolean = !SQLConf.get.legacyStatisticalAggregate
```

Comment:
so `nullOnDivideByZero` is coming from the config at execution time, and not from the aggregate function at plan time, which is different than Spark. Theoretically this should be fine, and it tests correctly in a shell against a distributed setup. I am just a little worried and wanted another pair of eyes to confirm (@revans2, @jlowe)

Reply:
+1 and thanks for the feedback @revans2.